src/main/java/witchinggadgets/common/items/armor/ItemPrimordialArmor.java

```diff
@@ -124,6 +124,10 @@ public void onLivingUpdateEvent(LivingUpdateEvent event) {
     public void onArmorTick(World world, EntityPlayer player, ItemStack stack) {
         if (!world.isRemote && stack.isItemDamaged() && player.ticksExisted % 20 == 0) stack.damageItem(-1, player);
 
+        if (player.isSneaking() && WGKeyHandler.jumpKey.isPressed() && !player.worldObj.isRemote) {
```

Comment:
Seems like this changes crashing player with internal error from server, when primordial armor is equipped. Stacktrace: https://mclo.gs/MKvS6Ui

Reply:
Yes , I missed this one when I fixed the other trying to run Client code on the Server error.

Should be an easy fix.